packages/grafana-ui/src/types/plugin.ts

```diff
@@ -49,7 +51,9 @@ export interface ExploreDataSourceApi<TQuery extends DataQuery = DataQuery> exte
 
 export interface QueryEditorProps<DSType extends DataSourceApi, TQuery extends DataQuery> {
   datasource: DSType;
+  panel?: PanelModel;
```

Comment:
Don’t want plugins to have access to the panel model , is there a reason for this and why is it optional?